### ISSUE TITLE
Easier translation of you plugin

### ISSUE THREAD

**Issue by xXSchrandXx** [OnlyNoobDied/GadgetsMenu#648]
**Is your feature request related to a problem? Please describe.**
Not realy, but the translation of you plugin is very hard...
If there is one mistake in the YAML the hole file does reset. (So everything again)

**Describe the solution you'd like**
A real translation page where everybody can suggest a translation. (like [Essentials](http://translate2.ess3.net/project/essentials-2) has with [Crowdin](https://crowdin.com/createproject))

**Describe alternatives you've considered**
Do save the messed up file so someone does not have to recreate his YAML.

**Additional context**
Some text I got for the german translation (I deleted some Strings):
[GadgetsMenu German Translation.zip](https://github.com/OnlyNoobDied/GadgetsMenu/files/2737890/GadgetsMenu.German.Translation.zip)

**Comment by xXSchrandXx** [comment]
Even [Minecraft](https://crowdin.com/project/minecraft/de#) does use Crowdin.

**Comment by Matocolotoe** [comment]
> If there is one mistake in the YAML the hole file does reset. (So everything again)

This is related to formatting errors, not language things.

**Comment by xXSchrandXx** [comment]
> This is related to formatting errors, not language things.

Even than, could you rename the wrong file to something like 'wrong.mainmenu.yml'.
Maybe write the error message in the last rows, too (So someone knows wich line he need to fix).

**Comment by Matocolotoe** [comment]
yeah like plotsquared does with filename.yml_broken, that'd indeed be nice

**Comment by OnlyNoobDied** [comment]
> yeah like plotsquared does with filename.yml_broken, that'd indeed be nice

Trying to make that but fails

**Comment by OnlyNoobDied** [comment]
I have added file backup feature when the file has broken. I think this would help a lot for the translation.
